changelog/next/features/4590--JSON-size-increase.md

```diff
@@ -0,0 +1 @@
+The JSON parser is now able to handle extremely large events.
```

Comment:
Going over this once more, I noticed that we already were able to do this, but just for NDJSON/GELF. If you want to make this more accurate, you could write something like "The JSON parser is now able to also handle very large events when not using the NDJSON mode.", or something like that.